Dockerfile

```
@@ -1,147 +1,118 @@
-# Set runtime environment variables
 ENV LANG=C.UTF-8 \
-    HOME=/app
+    HOME=/app \
+    MIX_ENV=prod
+
+ARG WANDERER_NOTIFIER_API_TOKEN
+ENV WANDERER_NOTIFIER_API_TOKEN=${WANDERER_NOTIFIER_API_TOKEN}
 
-# Install runtime dependencies (minimal set)
+# Update and install runtime packages
 RUN apt-get update -y && \
```

Hadolint (2.12.0)

[info] Delete the apt-get lists after installing something

(DL3009)